src/components/CollectionsPage/OptionsDropdown/OptionsDropdown.module.css

```diff
@@ -0,0 +1,24 @@
+.OptionsDropdown:global(.ui.dropdown) {
+  position: absolute;
+  top: 5px;
+  right: 10px;
+  z-index: 10000000;
+  display: inline-block;
+}
+
+.OptionsDropdown:global(.ui.dropdown) > :global(.dropdown.icon):before {
+  content: '…' !important;
+  color: black !important;
+}
+
+.OptionsDropdown:global(.ui.dropdown) > :global(.dropdown.icon) {
+  padding: 0 3px;
+  line-height: 2px;
+  font-size: 24px;
+  height: 0.7em;
+}
+
+.OptionsDropdown :global(.dropdown.icon):hover {
+  background-color: rgba(255, 255, 255, 0.08);
```

Comment:
You think this might be relevant enough for a css var (`rgba(255, 255, 255, 0.08)`)? it's repeated in three places

Reply:
Done!